app/services/license_and_rights_statements_setter.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+# Sets the license and rights statements (copyright, use, and reproduction statements)
+class LicenseAndRightsStatementsSetter
+  # @param [Ability] ability ability representing current user and perms
+  # @param [String] druid a druid identifier
+  # @param [String] copyright_statement copyright statement text
+  # @param [String] license license text
```

Comment:
this is really the code, right?  Can the description be fixed and even better, can we use a URI instead? That will make it more future-proof.

Reply:
@jcoyne I'm happy to update the YARD doc to be clearer. I'll push that in a minute.

Given the size of this PR already (> 1Kloc), I'd like to defer the future-proofing/URI-ization to a follow-on PR if you can be OK with that. (This is in response to the other 3-4 spots where you suggest future-proofing, which I approve of but first would like to land this beast.)

Reply:
Pushed.